api/tests/certification/enrolment/acceptance/application/session-mass-import-route_test.js

```diff
@@ -248,7 +248,7 @@ describe('Acceptance | Controller | Session | session-mass-import-route', functi
             date: '2023-01-01',
             time: '11:00',
             accessCode: 'accessCode',
-            supervisorPassword: 'KV2CP',
+            invigilatorPassword: 'KV2CPA',
```

Comment:
Vu que finalement pas de A ^^'
```suggestion
            invigilatorPassword: 'KV2CPB',
```